bakerydemo/breads/models.py

```diff
@@ -94,26 +94,27 @@ class BreadsIndexPage(BasePageFieldsMixin, Page):
 
     subpage_types = ['BreadPage']
 
-    def get_context(self, request):
-        context = super(BreadsIndexPage, self).get_context(request)
-
-        # Get the full unpaginated listing of resource pages as a queryset -
-        # replace this with your own query as appropriate
-        all_resources = self.get_children().live()
-
-        paginator = Paginator(all_resources, 5)  # Show 5 resources per page
+    @property
+    def breads(self):
```

Comment:
No problem to change the name to `get_breads`. But removing the `@property` raises an error `object of type 'method' has no len()` on the pagintor. I'm not sure why that's happening. Would welcome suggestions on how to resolve.